eblocker-icapserver/src/main/java/org/eblocker/server/icap/transaction/processor/RemoveTrackingParametersProcessor.java

```diff
@@ -0,0 +1,63 @@
+package org.eblocker.server.icap.transaction.processor;
+
+import com.google.inject.Singleton;
+import io.netty.handler.codec.http.FullHttpRequest;
+import org.eblocker.server.icap.transaction.Transaction;
+import org.eblocker.server.icap.transaction.TransactionProcessor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+
+@Singleton
+public class RemoveTrackingParametersProcessor implements TransactionProcessor {
+
+    private static final Logger log = LoggerFactory.getLogger(RemoveTrackingParametersProcessor.class);
+
+    private static final List<String> TRACKING_PARAMS = List.of("gclid", "msclkid", "fbclid", "utm_campaign", "utm_term", "utm_medium", "utm_source", "utm_content", "fb_action_ids", "fb_action_types", "fb_source", "fb_ref", "ga_source", "ga_medium",
```

Comment:
Better use a Set, so contains() scales better if the number of parameters grows?